test/integration/long/execution-profile-tests.js

```diff
@@ -0,0 +1,93 @@
+"use strict";
+var assert = require('assert');
+var util = require('util');
+
+var helper = require('../../test-helper.js');
+var Client = require('../../../lib/client.js');
+var types = require('../../../lib/types');
+var utils = require('../../../lib/utils.js');
+var loadBalancing = require('../../../lib/policies/load-balancing.js');
+var DCAwareRoundRobinPolicy = loadBalancing.DCAwareRoundRobinPolicy;
+var WhiteListPolicy = loadBalancing.WhiteListPolicy;
+var ExecutionProfile = require('../../../lib/execution-profile.js').ExecutionProfile;
+
+describe('ProfileManager', function() {
+  this.timeout(120000);
+  before(helper.ccmHelper.start('2:2:2'));
+  after(helper.ccmHelper.remove);
+
+  var dc1Profile = new ExecutionProfile('default', {
+    loadBalancing: new DCAwareRoundRobinPolicy('dc1')
+  });
+  var dc2Profile = new ExecutionProfile('DC2', {
+    loadBalancing: new DCAwareRoundRobinPolicy('dc2')
+  });
+  var wlProfile = new ExecutionProfile('whitelist', {
+    loadBalancing: new WhiteListPolicy(new DCAwareRoundRobinPolicy('dc3'), [helper.ipPrefix + '6' + ":9042"])
+  });
+  var emptyProfile = new ExecutionProfile('empty');
+
+  var profiles = [dc1Profile, dc2Profile, wlProfile, emptyProfile];
+
+  function newInstance(options) {
+    options = options || {};
+    options = utils.extend({
+      profiles: profiles
```

Comment:
We should use different profile instances for different clients. It works here because the cluster is the same, but `lbp.init()` is being called multiple times (per each client).

Something like: `profiles: createProfiles()`